lib/ts/recipe/session/types.ts

```diff
@@ -133,12 +146,25 @@ export type TypeNormalisedInput = {
     sessionExpiredStatusCode: number;
     errorHandlers: NormalisedErrorHandlers;
     antiCsrf: "VIA_TOKEN" | "VIA_CUSTOM_HEADER" | "NONE";
+    jwt:
+        | { enable: true; propertyNameInAccessTokenPayload: string }
+        | { enable: false; propertyNameInAccessTokenPayload: string };
```

Comment:
Can you change this to `{ enable: boolean; propertyNameInAccessTokenPayload: string }` (only in normalised input)?

Reply:
Done